docs/examples/ref.rb

```diff
@@ -17,18 +17,14 @@
 
 aws_security_group 'ref-sg1' do
   vpc 'ref-vpc'
-  inbound_rules [ { ports: 22, protocol: :tcp, sources: [ '0.0.0.0/0' ] } ]
+  inbound_rules '0.0.0.0/0' => 22
 end
 
 aws_security_group 'ref-sg2' do
   vpc 'ref-vpc'
 
-  inbound_rules [
-    {:ports => 2223, :protocol => :tcp, :sources => ['ref-sg1'] }
-  ]
-  outbound_rules [
-    {:ports => 2223, :protocol => :tcp, :destinations => ['ref-sg1'] }
-  ]
+  inbound_rules 'ref-sg1' => 2224
+  outbound_rules 2224 => 'ref-sg1'
```

Comment:
How is the protocol specified now? I'm assuming it defaults to tcp, which is fine, as long as it can be overridden in some way still?